src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(el => el.id === activeTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                className={cn({
+                  'is-active':
+                    activeTabId === tab.id ||
+                    (!tabs.some(t => t.id === activeTabId) &&
+                      tab.id === tabs[0]?.id),
+                })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (tab.id !== activeTabId) {
+                      onTabSelected(tab.id);
+                    }
+                  }}
```

Comment:
Consider will be better if you move this logic to the handler function and use it here